lib/ansible/cli/__init__.py

```diff
@@ -40,6 +40,28 @@ def check_blocking_io():
 
 check_blocking_io()
 
+
+def check_encoding():
+    """Check file system encoding and locale to ensure UTF-8."""
+    import ansible.utils.locale as locale_utils
+    fs_enc = sys.getfilesystemencoding()
+    if fs_enc.lower() != 'utf-8':
+        raise SystemExit('ERROR: Ansible requires the system encoding to be UTF-8; Detected %s.' % fs_enc)
```

Comment:
I missed this one in my last review:

```suggestion
        raise SystemExit('ERROR: Ansible requires the filesystem encoding to be UTF-8; Detected %s.' % fs_enc)
```